### DISCUSSION TITLE
Wordpress to Gatsby with theme

### DISCUSSION THREAD

**Discussion by riddhivmj** [gatsbyjs/gatsby#31275] [Ideas / Feature Requests]
Hi Team,

I have a wordpress website which is being builded using Salient theme. I have used WP Bakery page builder to create entire website and now i want to make my website headless with Gatsby. I understand gatsby fetches wordpress CMS contents using WP Rest API. But the page contents which is builded using page builder is saved in the form of theme shortcodes in database. Which can be only rendered by theme modules. Since the WP Rest API also returns the shortcode, how can i build a gatsby website using this ?

**Comment by web86** [comment]
Hi, I'm also looking for the solution, did you find something already?

**Reply by web86** [reply]
I've found usefull tutorial here https://tbntdima.com/blog/wordpress-to-gatsby-migration/
